.php-cs-fixer.dist.php

```diff
@@ -29,5 +29,6 @@
         ],
         'single_line_throw' => false, //override Symfony
         'yoda_style' => false, //override Symfony
+        'phpdoc_to_comment' => false, //incorrectly changes @var tags
```

Comment:
It might be better to just exclude the specific tag we want to keep?

```suggestion
        'phpdoc_to_comment' => [ // Keep as a phpdoc if it contains the `@var` annotation
            'ignored_tags' => ['var'],
        ],
```

Reply:
Ah, did not known about that option, updated!